packages/engine/src/networking/serialization/DataWriter.ts

```diff
@@ -81,10 +86,99 @@ export const writeVector4 = (vector4: Vector4SoA) => (v: ViewCursor, entity: Ent
   return (changeMask > 0 && writeChangeMask(changeMask)) || rewind()
 }
 
+// Writes a compressed Quaternion value to the network stream. This function uses the "smallest three"
+// method, which is well summarized here: http://gafferongames.com/networked-physics/snapshot-compression/
+// Started with this code as baseline: https://gist.github.com/StagPoint/bb7edf61c2e97ce54e3e4561627f6582
+export const writeComporessedRotation = (vector4: Vector4SoA) => (v: ViewCursor, entity: Entity) => {
```

Comment:
typo, should be spelled `writeCompressedRotation`

Reply:
Fixed.